src/scripts/keyboardNavigation.js

```diff
@@ -116,6 +132,11 @@ export function enable() {
             return;
         }
 
+        // Ignore Media Keys for non-TV platform having MediaSession API
+        if (!layoutManager.tv && isMediaKey(key) && hasMediaSession) {
```

Comment:
Overlooked this. We can activate TV layout on the PC. We should test the browser "type".
```suggestion
        if (!browser.tv && isMediaKey(key) && hasMediaSession) {
```